src/interfaces/element-props.ts

```diff
@@ -1,9 +1,32 @@
+export const enum Phases {
+  Gas = 'Gas',
+  Liquid = 'Liquid',
+  Solid = 'Solid',
+}
+
+export const enum Categories {
+  DiatomicNonmetal = 'diatomic nonmetal',
+  NobleGas = 'noble gas',
+  AlkaliMetal = 'alkali metal',
+  AlkalineEarthMetal = 'alkaline earth metal',
+  Metalloid = 'metalloid',
+  PolyatomicNonmetal = 'polyatomic nonmetal',
+  PostTransitionMetal = 'post-transition metal',
+  TransitionMetal = 'transition metal',
+  Lanthanide = 'lanthanide',
+  Actinide = 'actinide',
+}
+
+export type Category = keyof typeof Categories;
+
+export type Phase = keyof typeof Phases;
+
 export interface ElementProps {
   name: string;
   appearance?: string;
   atomicMass: number;
   boil?: number;
-  category: string;
+  category: Category | string;
```

Comment:
Does Category does not cover all types? If yes why is string type needed?

Reply:
no ,Besides a set of values for category , It has some string that defines the possible categories with a little description, so we needed to define as though. see Astatine element for example